### ISSUE TITLE
[XAML Controls Gallery(debug)->Layout->ItemsRepeater] : Color button names are not properly visible in 'animated scrolling and content display' pane in HC black and white modes

### ISSUE THREAD

**Issue by v-amnat** [microsoft/Xaml-Controls-Gallery#680]
**Tags**
Accessibility, A11yCT, A11yMAS, A11yMediumImpact, MAS4.3.1, SupportingthePlatform,
 A11ySev2, XAMLControlsGallery, WinUI, Win32, Wipro

**Test Environment**
1. XAML Controls Gallery(debug)  version - 2.0.21.0
2. Win 32/ OS build- 21343.rs_Prerelease.210320-1757

**Steps to reproduce the bug**

1. Launch the XAML Controls Gallery (Debug) App
2. Go to the Layout-> ItemsRepeater.
3. Hit on Tab key to traverse the "ItemsRepeater(Carousals)".
4. Observe the color button names in 'animated scrolling and content display' pane.

**Actual behavior**
In Hc White and black modes color button names are not visible properly.

**Expected**
All content on the page should remain visible/distinguishable in Windows high contrast modes. Typically, these problems are caused by using icons or images when CSS or text could be used, or by not using semantic code to build interactive components. This is a valuable resource for high contrast mode fixes. 

**User Impact** 
When content or components disappear in high contrast modes or they don’t appear as interactive elements, customers who rely on that feature are blocked from information and functionality of the full user experience within the site/app. 

**MAS Reference**
MAS 4.3.1 - No Disruption of Accessibility Features 

**Attachments**
[Colors content display.zip](https://github.com/microsoft/Xaml-Controls-Gallery/files/6310531/Colors.content.display.zip)

**Comment by chigy** [comment]
@anawishnoff fixed this
